src/opset/Conv_11.c

```diff
@@ -22,133 +22,44 @@
 #include <connx/connx.h>
 
 /*{% for DTYPE, TYPE in loop_types(FLOAT32, FLOAT64) %}*/
+#define MAX(a, b) ((a) > (b) ? (a) : (b))
+#define MIN(a, b) ((a) < (b) ? (a) : (b))
+
+#include <stdio.h>
 // clang-format off
-static void _conv_{{ DTYPE | to_name }}({{TYPE}}* Y_flatten, {{TYPE}}* X_flatten, int32_t feature_dim,
-                                int32_t* feature_shape, connx_Iterator* x_iter, {{TYPE}}* W_flatten,
-                                int32_t* kernel_shape, int32_t* dilations) {
+static void conv_1d_{{DTYPE}}({{TYPE}}* Y_flatten, int32_t* Y_shape, {{TYPE}}* X_flatten, int32_t* X_shape, {{TYPE}}* W_flatten, int32_t* W_shape, int32_t* pads, int32_t* dilations, int32_t* strides) {
```

Comment:
Although We used `clang-format off` for template, Follow clang-format's linebreak is look better.
1. Remove `//clang-format off` line
2. Run clang-format
3. Fix malformed template and re-attach `//clang-format off` line

Reply:
Done!